automation/.gitignore

```diff
@@ -0,0 +1,3 @@
+.tox
```

Comment:
.gitignore file should be in root directory.

Reply:
done